scrapers/ar/events.py

```diff
@@ -110,13 +111,19 @@ def scrape(self, start=None, end=None):
 
             yield event
 
-    def scrape_referred_bills(self, event, url):
-        page = self.get(url).content
-        page = lxml.html.fromstring(page)
-        page.make_links_absolute(url)
 
-        if page.xpath("//div[contains(@class,'billSubtitle')]"):
-            agenda = event.add_agenda_item("Referred Bills")
+class Agenda(PdfPage):
+    def process_page(self):
+        # Search for bill ids in text
+        matches = re.findall(
+            r"((SB|HB|SR|HR)\d+)",
+            self.text,
+        )
+
+        # Remove duplicates
+        matches = list(set(matches))
 
-            for row in page.xpath("//div[contains(@class,'measureTitle')]/a/text()"):
-                agenda.add_bill(row.strip())
+        # Yield results, ignoring an a duplicate part of the regex
+        for m, _ in matches:
+            # Add in space between letters and numbers
+            yield m[:2] + " " + m[2:]
```

Comment:
Maybe `f"{m[:2]} {m[2:]}"` instead?

Reply:
Yes that would be easier to read - fixed with https://github.com/openstates/openstates-scrapers/pull/4518/commits/f82e7b161ceff5f7ec1501a267fac7ca0df5d36b Thanks :smile: